apps/web/pages/checkout.vue

```diff
@@ -36,21 +36,8 @@
         <UiDivider class="w-screen md:w-auto -mx-4 md:mx-0" />
         <CheckoutPayment :payment-methods="paymentMethods" @update:active-payment="handlePaymentMethodUpdate($event)" />
         <UiDivider class="w-screen md:w-auto -mx-4 md:mx-0 mb-10" />
-      </div>
-      <OrderSummary v-if="cart" :cart="cart" class="col-span-5 md:sticky md:top-20 h-fit">
-        <SfButton
-          type="submit"
-          @click="order"
-          :disabled="createOrderLoading"
-          size="lg"
-          class="w-full mb-4 md:mb-0 cursor-pointer"
-        >
-          <SfLoaderCircular v-if="createOrderLoading" class="flex justify-center items-center" size="sm" />
-          <span v-else>
-            {{ $t('placeOrder') }}
-          </span>
-        </SfButton>
-        <p class="text-sm text-center mt-4 pb-4 md:pb-0">
+        <div class="text-sm mx-4 md:pb-0">
```

Comment:
Checkbox and Label should be wrapped into a div and then use class items-center to align the box with the text